Main.py

```diff
@@ -0,0 +1 @@
+print('Hello, world!')
```

Comment:
All work concerning the first lab should be located in the `lab_1_classify_by_unigrams` folder.
Please, remove extra files.

Reply:
I removed all extra files

Reply:
Done

Reply:
I can't delete the file. Could you explain me how I can do it? I press the button 'Delete file' and it remains on its place

Reply:
You should firstly delete the file locally, then register this change to the git via `git add <path to the deleted file>`, then you commit the change via `git commit <message>` and, finally, push it to the remote fork via `git push`. To see the path to the deleted file use `git status`, it should indicate that you have deleted the file tracked. 
If you have any more questions, please consult student chat and enclose pictures.

Reply:
Done